_posts/2019-08-05-world-moveit-day-2019.md

```diff
@@ -58,7 +58,7 @@ Please state your intent to join or host the international event on [this form](
 - USA, CA, Union City, [Vicarious](https://www.vicarious.com/)
   - Organizer: Peter So
 - USA, CA, San Jose, [Fetch / ROS by the Bay](https://fetchrobotics.com/)
```

Comment:
```suggestion
- USA, CA, San Jose, [Fetch Robotics](https://fetchrobotics.com/)
```

Reply:
Could we also just change this to "Fetch Robotics"?

The plan is to host "World MoveIt Day" during business hours, and then host "ROS by the Bay" https://www.meetup.com/ROS-by-the-Bay/ in the evening with @jacobperron

Reply:
FYI @cjds I hadn't see this PR but happened to send you that email today. Looks like someone is already on top of it.